resources/access.rb

```diff
@@ -45,11 +45,15 @@
       }
       action :nothing
       delayed_action :create
-      notifies new_resource.notification, postgresql_service
+      notifies :trigger, config_resource, :immediately
     end
   end
 end
 
+action :trigger do
+  new_resource.updated_by_last_action(true)
```

Comment:
This (and the ident resource) is throwing a FC085 warning. Do we either ignore this error or find another way of doing this, and I don't know how to in this particular case(!0